setup.cfg

```diff
@@ -1,6 +1,75 @@
-[nosetests]
-nocapture=1
-cover-package=pyramid_nacl_session
-cover-erase=1
-cover-branches=1
-cover-min-percentage=100
+[metadata]
+name = pyramid_nacl_session
+version = 1.0.0
+description = Encrypted session cookie serializer for Pyramid
+long_description = file: README.rst, CHANGES.txt
+long_description_content_type = text/x-rst
+keywords = pyramid sessions crypography security
+license = BSD-derived (http://www.repoze.org/LICENSE.txt)
+classifiers =
+    Development Status :: 6 - Mature
+    Environment :: Web Environment
+    Framework :: Pyramid
+    Intended Audience :: Developers
+    License :: Repoze Public License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.6
+    Programming Language :: Python :: 2.7
```

Comment:
at least 2.6 I doubt we are testing any longer

Reply:
oh yeah it's excluded in python_requires, so should be dropped

Reply:
ARGH, I thought I had double checked those... lol

Reply:
I have no idea what you are talking about...